modules/openy_features/openy_location/modules/openy_loc_facility/config/install/field.field.node.facility.field_location_area.yml

```diff
@@ -5,10 +5,6 @@ dependencies:
     - field.storage.node.field_location_area
     - node.type.facility
     - taxonomy.vocabulary.area
-third_party_settings:
-  datalayer:
-    expose: 1
-    label: field_location_area
```

Comment:
Sorry for not clarifying. I had removed this because exposing the entity reference fields is what is causing the fatal error. https://www.drupal.org/node/2857270 "Entity reference field selected causing Fatal Error".